OpenRA.Mods.Common/Traits/Player/ClassicProductionQueue.cs

```diff
@@ -54,6 +54,9 @@ protected override void Tick(Actor self)
 			isActive = false;
 			foreach (var x in self.World.ActorsWithTrait<Production>())
 			{
+				if (x.Trait.IsTraitDisabled)
```

Comment:
I now wonder if `ProductionQueue.Tick()` also needs such a check.

Reply:
I now made it so that `ProductionQueue.Tick` cancels current production incl. queued items completely when all production traits are disabled, and at least doesn't tick the queue when they are paused. Hope that works as intended...